server/sonar-server/src/main/resources/org/sonar/server/usertoken/ws/search-example.json

```diff
@@ -0,0 +1,19 @@
+{
+  "userTokens": [
+    {
+      "login": "grace.hopper",
```

Comment:
it's not/it won't be possible to return tokens on different users, so repeating the login is misleading. I would move the field "login" to the root node.